xbmc/cores/VideoPlayer/DVDOverlayContainer.cpp

```diff
@@ -115,10 +115,14 @@ void CDVDOverlayContainer::Flush()
   std::unique_lock<CCriticalSection> lock(*this);
 
   // Flush only the overlays marked as flushable
-  m_overlays.erase(
-      std::remove_if(m_overlays.begin(), m_overlays.end(),
-                     [](CDVDOverlay* ov) { return ov->IsOverlayContainerFlushable(); }),
-      m_overlays.end());
+  m_overlays.erase(std::remove_if(m_overlays.begin(), m_overlays.end(),
+                                  [](CDVDOverlay* ov) {
+                                    bool isFlushable = ov->IsOverlayContainerFlushable();
```

Comment:
Is:
```suggestion
                                    const bool isFlushable = ov->IsOverlayContainerFlushable();
```
of any value here?

Reply:
"const" in this kind of situation is very useful to avoid some bugs at the C++ code level, but it doesn't change the code generated at the compiler level.